platforms/android/build_java_shared_aar.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python
```

Comment:
```suggestion
#!/usr/bin/env python3
```

I'd suggest using _pathlib_ for all path/file related operations (https://docs.python.org/3.8/library/pathlib.html#correspondence-to-tools-in-the-os-module).
We support python 3.8+, do we?